.gitlab-ci.yml

```diff
@@ -88,6 +88,10 @@ Base:
           # see COMPOSER-918
           # - aws/rhel-8.4-aarch64
         INTERNAL_NETWORK: ["true"]
+  artifacts:
+    paths:
```

Comment:
What's the reason behind uploading the repo file in the base tests? I would expect it in the RPM build stage.

Reply:
Oh, the journal is now not saved for base tests.

Reply:
> What's the reason behind uploading the repo file in the base tests? I would expect it in the RPM build stage.

The repo file is created via `setup_repo()` in `deploy.sh`, which isn't executed during rpmbuild.